src/services/api/FirebaseApi.ts

```diff
@@ -654,7 +654,7 @@ export class FirebaseApi implements ServiceApi {
       where("subject", "==", course.subject),
       where("curriculum", "==", course.curriculum)
     );
-    const queryResult = await this.getDocsFromOffline(q);
+    const queryResult = await getDocs(q);
```

Comment:
Every time it will fetch from firebase server but we need to fetch from cache also if it incase not found in cache then fetch from server